src/PhappCommandBase.php

```diff
@@ -206,6 +245,7 @@ protected function invokeManifestCommand($command_name) {
   protected function invokeManifestCommandAtDirectory($command_name, $directory) {
     $collection = $this->collectionBuilder();
     $manifest = PhappManifest::getInstance($directory);
+    $this->setSubappManifest($directory, $manifest);
```

Comment:
This getter/setter approach seems weird when all we need is passing $manifest in the getPhappEnvironmentVariables() invcocation 5 lines below. Line 257. Let's add it as optional parameter and use $this->manifest if nothing is passed. If the manifest is passed, use that.